src/pages/Group/Index.js

```diff
@@ -607,17 +609,17 @@ class Main extends PureComponent {
             <div className={styles.connect_Box}>
               <div className={styles.connect_Boxs}>
                 <div>使用内存</div>
-                <div>{resources.memory || 0}M</div>
+                <div>{`${sourceUtil.unit(resources.memory || 0, 'MB')}`}</div>
               </div>
               <div className={styles.connect_Boxs}>
                 <div>使用CPU</div>
-                <div>{resources.cpu || 0}M</div>
+                <div>{(resources.cpu && resources.cpu / 1000) || 0}核</div>
               </div>
             </div>
             <div className={styles.connect_Box}>
               <div className={styles.connect_Boxs}>
                 <div>使用磁盘</div>
-                <div>{resources.disk || 0}MB</div>
+                <div>{`${sourceUtil.unit(resources.disk || 0, 'MB')}`}</div>
```

Comment:
Same problem as memory.

Reply:
Intelligent processing unit of data sourceUtil.unit methods